TVGuide/Utils/StringManager.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+class StringManager {
+    
+    static func getStrFromArrOfStr(_ strArr: [String]?) -> String? {
+
+        var resultStr = ""
+
+        guard let sa = strArr else {
```

Comment:
Trata de que los nombres sean mas demostrativos, al punto que si yo leo el for, entiendo que esta haciendo sin necesidad de leer el resto del codigo. Por que sa y s no es muy descriptivo si no entiendo el contexto